app/models/party.rb

```diff
@@ -0,0 +1,38 @@
+class Party < ActiveRecord::Base
+  has_many :rsvps
+  has_many :users, through: :rsvps
+  has_many :unregistered_email
+
+  def combine_nonos
+    @users = self.users
+    @combined_nonos = []
+    @users.each do |user|
```

Comment:
You probably don't need to do the nested loop here. You can use array union. Consider...

```
2.1.4 :004 > a = [1, 2]
 => [1, 2] 
2.1.4 :005 > b = [2, 3]
 => [2, 3] 
2.1.4 :006 > a | b
 => [1, 2, 3] 
```